src/app/components/RecentTrades/RecentSwapRow.tsx

```diff
@@ -22,11 +23,32 @@ export const RecentSwapRow: React.FC<RecentSwapRowProps> = ({
     () => (isOddRow ? 'tw-bg-gray-3' : 'tw-bg-gray-1'),
     [isOddRow],
   );
-  const isBuy = useMemo(() => row.type === SwapsType.BUY, [row.type]);
+  const isBuy = useMemo(
+    () =>
+      compareAddress(
+        row._fromToken.id,
+        baseAssetDetails.getTokenContractAddress(),
+      ),
+    [baseAssetDetails, row._fromToken.id],
+  );
+
+  const size = useMemo(() => (isBuy ? row._return : row._amount), [
+    isBuy,
+    row._amount,
+    row._return,
+  ]);
+
+  const price = useMemo(
+    () =>
+      isBuy
+        ? Number(row._amount) / Number(row._return)
+        : Number(row._return) / Number(row._amount),
+    [isBuy, row._amount, row._return],
+  );
```

Comment:
The logic here is not correct - I paid `100 XUSD` and received `0,00476816 BPRO`.
<img width="1768" alt="image" src="https://user-images.githubusercontent.com/14878713/193834256-eb46a4ca-f3d2-407e-b9a1-9b97cf6987c4.png">

This is what I see on `testnet` (and it's correct):
<img width="340" alt="image" src="https://user-images.githubusercontent.com/14878713/193834400-92b63ec5-a43a-430a-8355-1ed93ee3d7cc.png">

And this is what I see on `local`: 
<img width="340" alt="image" src="https://user-images.githubusercontent.com/14878713/193834507-0f42c54a-dec2-4e57-89e5-061d96d01f0f.png">